index.html

```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
+                        of my
+                        work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                        contact
+                        me.</p>
+                    <p class="connect">Let's connect</p>
+                </div>
+                <div class="about-social">
+                    <ul>
+                        <li><a href="#"><img src="./images/github.png"></a></li>
+                        <li><a href="#"><img src="./images/linkedin.png"></a></li>
+                        <li><a href="#"><img src="./images/hand.png"></a></li>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/m.png"></a></li>
+                    </ul>
+                </div>
+                <div class="about-resume">
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+            </div>
             <div class="details">
                 <ul>
                     <div class="drop-title">
                         <h3>Languages</h3>
                         <img src="./images/down-arrow.png">
                     </div>
-                    <li>
+                    <li class="js-tag">
                         <div class="list-language">
                             <img src="./images/js.png" alt="">
                             <p>javascript</p>
                         </div>
                     </li>
-                    <li>
+                    <li class="html-tag">
                         <div class="list-language">
                             <img src="./images/html.png" alt="">
                             <p>Html</p>
                         </div>
                     </li>
-                    <li>
+                    <li class="css-tag">
                         <div class="list-language">
                             <img src="./images/css.png" alt="">
                             <p>CSS</p>
                         </div>
                     </li>
+                    <div id="lang-tag-new">
+                        <li>
+                            <div class="list-language">
+                                <img src="./images/js.png" alt="">
+                                <p>javascript</p>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="list-language">
+                                <img src="./images/html.png" alt="">
+                                <p>Html</p>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="list-language">
+                                <img src="./images/css.png" alt="">
+                                <p>CSS</p>
+                            </div>
+                        </li>
+                    </div>
```

Comment:
- [ ] Kindly add some spacing between these `div`s to match the template 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215290937-a53fa100-ecfc-4aeb-9c41-dace7ee86ce2.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215290941-f9ea813f-f2ac-4fc6-a9db-3a5a0b622382.png)